integrations/zoom.js

```diff
@@ -87,7 +87,11 @@ module.exports = async function(robot, kredits) {
     const payload = data.payload;
     const object = payload.object;
 
-    if (eventName === 'meeting.ended') {
+
+    if (eventName === 'meeting.ended' && (
+        !process.env.KREDITS_ZOOM_MEETING_WHITELIST ||
+          process.env.KREDITS_ZOOM_MEETING_WHITELIST.split(',').includes(object.id)
```

Comment:
While we're adding untested code, how about this? :smile: 

```suggestion
        process.env.KREDITS_ZOOM_MEETING_WHITELIST?.split(',').includes(object.id)
```

Reply:
@bumi Did you see this?

Reply:
actually that does not work for me:
```
process.env.KREDITS_ZOOM_MEETING_WHITELIST?.split(',').includes(object.id)
SyntaxError: Unexpected token '.'
```
(the `.` after the `?`)

how does that `?` work?

Reply:
It's called optional chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

But in this case, it would change the existing logic. It would not call `handleZoomMeetingEnded()` if there was no `KREDITS_ZOOM_MEETING_WHITELIST` environment variable defined, while the existing code would still call the function.

Reply:
True. I missed the functional change.

Reply:
ah, thanks. good to know.